test/ctia/flows/hooks/kafka_event_hook_test.clj

```diff
@@ -0,0 +1,167 @@
+                  test-helpers/fixture-properties:events-enabled
+                  test-helpers/fixture-ctia
+                  test-helpers/fixture-allow-all-auth]))
+
+(deftest ^:integration test-events-topic
+  (testing "Events are published to kafka topic"
+    (let [results (atom [])
+          finish-signal (CountDownLatch. 3)
+          {:keys [channel-name] :as kafka-config}
+          (get-in @properties [:ctia :hook :kafka])
+          consumer-map
+          (lk/subscribe
+           (fn test-events-kafka-topic-fn [ev]
+             (let [v (:value ev)]
+               (swap! results conj (parse-string v true))
+               (.countDown finish-signal)))
+           30000)
+          ;; await rebalance
+          _ (Thread/sleep 5000)]
+      (let [{{judgement-1-long-id :id} :parsed-body
+             judgement-1-status :status
+             :as judgement-1}
+            (post "ctia/judgement"
+                  :body {:observable {:value "1.2.3.4"
+                                      :type "ip"}
+                         :disposition 1
+                         :source "source"
+                         :tlp "green"
+                         :priority 100
+                         :severity "High"
+                         :confidence "Low"
+                         :valid_time {:start_time "2016-02-11T00:40:48.212-00:00"}})
+
+            judgement-1-id
+            (id/long-id->id judgement-1-long-id)
+
+            {{judgement-2-long-id :id} :parsed-body
+             judgement-2-status :status
+             :as judgement-2}
+            (post "ctia/judgement"
+                  :body {:observable {:value "1.2.3.4"
+                                      :type "ip"}
+                         :disposition 2
+                         :source "source"
+                         :tlp "green"
+                         :priority 100
+                         :severity "High"
+                         :confidence "Low"
+                         :valid_time {:start_time "2016-02-11T00:40:48.212-00:00"}})
+
+            judgement-2-id
+            (id/long-id->id judgement-2-long-id)
+
+            {{judgement-3-long-id :id} :parsed-body
+             judgement-3-status :status
+             :as judgement-3}
+            (post "ctia/judgement"
+                  :body {:observable {:value "1.2.3.4"
+                                      :type "ip"}
+                         :disposition 3
+                         :source "source"
+                         :tlp "green"
+                         :priority 100
+                         :severity "High"
+                         :confidence "Low"
+                         :valid_time {:start_time "2016-02-11T00:40:48.212-00:00"}})
+
+            judgement-3-id
+            (id/long-id->id judgement-3-long-id)]
+
+        (is (= 201 judgement-1-status))
+        (is (= 201 judgement-2-status))
+        (is (= 201 judgement-3-status))
+
+        (is (.await finish-signal 5 TimeUnit/SECONDS)
```

Comment:
kind of the same remark about waiting here, travis CI can be tricky about that, perhaps you would be safer by increasing that value a lot.